### PR TITLE
Fwx

### PR DESCRIPTION
**Pull request by earth2378**
**What changed? Why?**

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: apps/web/src/data/ecosystem.json [hunk 1]
```diff
@@ -4223,5 +4223,15 @@
...
-  }
+  },
+  {
+  "name": "FWX",
+  "tags": [
+      "defi"
+  ],
+  "description": " FWX is a permissionless perps DEX, any token can be permissionlessly listed to create their own onchain perps and lending markets. Trades are hedged in onchain DEXs to create zero risk environment for lenders.",
+  "url": "https://fwx.finance/",
+  "imageUrl": "/images/partners/your-project-logo.png"
```
Comment (wbnns) [inline apps/web/src/data/ecosystem.json:4234]: This should be a your logo name

File: apps/web/src/data/ecosystem.json [hunk 2]
```diff
@@ -4223,5 +4223,15 @@
...
     "url": "https://phantom.app/",
     "imageUrl": "/images/partners/phantom.svg"
-  }
+  },
+  {
+  "name": "FWX",
+  "tags": [
+      "defi"
+  ],
+  "description": " FWX is a permissionless perps DEX, any token can be permissionlessly listed to create their own onchain perps and lending markets. Trades are hedged in onchain DEXs to create zero risk environment for lenders.",
```
Comment (wbnns) [inline apps/web/src/data/ecosystem.json:4232]: Please remove extra space at beginning of description

**Review by wbnns — CHANGES_REQUESTED** [review]
@earth2378 

Thanks for this PR! Left a couple of nits in-line. Could you also please confirm what markets are available on Base?

**Comment by wbnns** [comment]
Closing due to inactivity

**Pull request closed without merging**